clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtil.groovy

```diff
@@ -118,6 +124,67 @@ class GCEUtil {
     }
   }
 
+  static Image getImageFromArtifact(Artifact artifact,
+                                    Compute compute,
+                                    Task task,
+                                    String phase,
+                                    SafeRetry safeRetry,
+                                    GoogleExecutorTraits executor) {
+    if (artifact.getType() != ArtifactUtils.GCE_IMAGE_TYPE) {
+      throw new GoogleOperationException("Artifact to deploy to GCE must be of type ${ArtifactUtils.GCE_IMAGE_TYPE}")
+    }
+
+    def reference = artifact.getReference()
+    task.updateStatus phase, "Looking up image $reference..."
+
+    def result = safeRetry.doRetry(
+      {
+        return compute.getRequestFactory()
+          .buildGetRequest(new GenericUrl(reference))
+          .setParser(new JsonObjectParser(JacksonFactory.getDefaultInstance()))
+          .execute()
+          .parseAs(Image)
+      },
+      "gce/image",
+      task,
+      RETRY_ERROR_CODES,
+      [],
+      [action: "get", phase: phase, operation: "compute.buildGetRequest.execute", (executor.TAG_SCOPE): executor.SCOPE_ZONAL],
```

Comment:
I think images are `global` so setting the call scope here as `SCOPE_ZONAL` seems off. I think there's a `SCOPE_GLOBAL`, otherwise define one.

Reply:
Good point, I'll fix